spec.yml

```diff
@@ -0,0 +1,33 @@
+requirements:
+  base:
+    image: debian:stretch
+    pkg-manager: apt
+  fsl:
+    version: 5.0.10
+  afni:
+    version: latest
+  miniconda:
+    conda_install: [python=3.7, nipype]
+
+nfm_test:
+  data:
+    type: workflow_path
```

Comment:
we should definitely write a spec definition somewhere, so we can enumerate things like this.

Reply:
@effigies - the intent of this `type` field here is to support things like datalad.

Reply:
Ah, gotcha. Though it seems you could do it with no change to semantics if you just make the directories submodules. You just change `git clone ...` to `datalad install -r ...`

Reply:
yup - both would be supported. 

if type==datalad, then testkraken will take care of getting the appropriate files. would have to figure out if we want autodiscover (based on finding a .datalad/.gitannex submodule, or simply pulling in an external repo to data path via datalad.